src/CodeGen_C.cpp

```diff
@@ -2550,13 +2562,19 @@ void CodeGen_C::visit(const Call *op) {
             rhs << "(" << print_type(op->type) << ")";
         }
         rhs << "(&" << struct_name << ((count > 0) ? "[0])" : ")");
-    } else if (op->is_intrinsic(Call::load_struct_member)) {
+    } else if (op->is_intrinsic(Call::load_typed_struct_member)) {
+        // Given an instance of a typed_struct, a definition of a typed_struct,
```

Comment:
Given all the confusion around a value that solely exists to communicate a type, this comment should probably spend a few words on what an "instance of a typed struct" is. I.e. a Handle type that was previously returned from one of the intrinsics that returns one.

Reply:
PTAL